### ISSUE TITLE
[HOLD on #11768] Poor app animation while opening report link from email 

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#15669]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open the app in android app and login with user A
2. Open the app on another device and login with user B
3. In user A device, keep the user B report open and minimize the app
4. From user B device, send message to user A
5. Open user A email and from email notification of recent user B message, open the new expensify link
6. Observe the app

## Expected Result:
App opens the report on link click without any shaky animation

## Actual Result:
When app opens the report using the link, app tries to open reports archive page first and within a second again tries to open report page which causes the poor animation as displayed in video

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.2.78-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/222943904-69db3584-63ae-4668-a578-2318faa6ec6e.mp4

https://user-images.githubusercontent.com/43996225/222943925-31243654-2f81-4573-ae6c-6a893eb9cfc3.mp4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @dhanashree-sawant
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1677948907449819

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @garrettmknight (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by garrettmknight** [comment]
Clarifying whether this is an improvement or a bug in slack thread.

**Comment by roryabraham** [comment]
I think this is a valid bug, but will most likely be fixed by the react-navigation reboot. Putting this on HOLD for https://github.com/Expensify/App/issues/11768

**Comment by garrettmknight** [comment]
Reassigning since I'm headed OOO - tracking issue for nav reboot is https://github.com/Expensify/App/issues/11768

**Comment by garrettmknight** [comment]
Dropping to weekly since we're still waiting on the reboot.

**Comment by kevinksullivan** [comment]
still on hold.

**Comment by kevinksullivan** [comment]
Held

**Comment by kevinksullivan** [comment]
Closing unless this comes up again.
